man/ronn.1.ronn

```diff
@@ -300,7 +300,7 @@ under a `man/` directory:
 
 ## BUGS
 
-**Ronn** is written in Ruby and depends on hpricot and rdiscount, extension
+**Ronn** is written in Ruby and depends on hpricot and kramdown, extension
```

Comment:
I appreciate that you updated the man page here, too!

Could you rebase this PR, though? I actually changed the man page to remove this BUGS item entirely, because IMHO these libs are easy enough to install now, this is not an issue any more.

Reply:
Rebased!